prinz/src/main/scala/pl/touk/nussknacker/prinz/mlflow/model/api/MLFSignatureNotFoundException.scala

```diff
@@ -0,0 +1,6 @@
+package pl.touk.nussknacker.prinz.mlflow.model.api
+
+import pl.touk.nussknacker.prinz.util.exceptions.PrinzException
+
+case class MLFSignatureNotFoundException(model: MLFRegisteredModel)
+  extends PrinzException(s"Cannot found MLFlow MLModel $model")
```

Comment:
Just to be perfect about the grammar - found -> find

Reply:
Will fix, thanks